test/test_cli.py

```diff
@@ -136,3 +139,66 @@ def test_should_find_root_package_when_root_used_project_name_used(self):
 
         # then
         self.assertEqual(test_module_src, resolve(res))
+
+    def test_should_run_workflow(self):
```

Comment:
When I read this test I wonder what's the difference between each workflow you run. Do they represent different situations?

Reply:
No, just different jobs and workflows. I check if when I run a workflow only this workflow is started, and when I start another one that only this other one is started.